build-system/tasks/storybook/index.js

```diff
@@ -0,0 +1,74 @@
+/**
+ * Copyright 2020 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+'use strict';
+
+const argv = require('minimist')(process.argv.slice(2));
+const {execScriptAsync} = require('../../common/exec');
+const {installPackages} = require('../../common/utils');
+
+let storybookArgs = '--quiet';
+if (argv.port) {
+  storybookArgs += ` -p ${argv.port}`;
+}
+
+/**
+ * Simple wrapper around the storybook start script
+ * for AMP components (HTML Environment)
+ */
+async function storybookAmp() {
+  // install storybook-specific modules
+  installPackages(__dirname);
+
+  execScriptAsync(
```

Comment:
This function starts the task in daemon mode. Are you sure this is what should happen? Why not use `exec`?

Also, if this is a long-running task that could benefit from a Ctrl + C handler to let the user easily exit the task, you could use this pattern:

https://github.com/ampproject/amphtml/blob/00a728570949e07236b5387d85f8cf1a75c1c157/build-system/tasks/visual-diff/index.js#L680-L700

Reply:
You're right, `exec` is sufficient, I thought it didn't support changing the `cwd`